lib/ethers.ex

```diff
@@ -153,6 +153,23 @@ defmodule Ethers do
     |> post_process(nil, :get_transaction)
   end
 
+  @doc """
+  Returns the native transaction (ETH) receipt by transaction hash.
+
+  ## Parameters
+  - tx_hash: Transaction hash which the transaction receipt is queried for.
+  - overrides:
+    - rpc_opts: Specific RPC options to specify for this request.
```

Comment:
Seems like I also forgot to put `rpc_client`option in `get_transaction/2` too

```suggestion
  - overrides:
    - rpc_client: The RPC module to use for this request (overrides default).
    - rpc_opts: Specific RPC options to specify for this request.
```

Reply:
Will do :)